docs/source/usrman/reference.rst

```diff
@@ -0,0 +1,9 @@
+.. _reference:
```

Comment:
I guess this file is a placeholder for future expansion?

Reply:
Not really. We could add some introductory text, but the idea is to dump in here all the automatically generated docs for the `mpi4py.MPI` module.  Note the new file `mpi4py.MPI.rst`, that one should be extended with narrative-style docs intermixed with class/method listings.

Reply:
I'm not going to expand it RIGHT NOW, but maybe in the near future. I want to keep it because this document reflects the package.submodule structure, and things (e.g. classes and functions) are categorized by functionality (although it could be done much better).